vm/acpi/src/sdt/devices.rs

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use super::helpers::*;
+use super::objects::*;
+use super::ops::*;
+
+pub struct Method {
+    pub name: [u8; 4],
+    pub sync_level: u8,
+    pub is_serialized: bool,
+    pub arg_count: u8,
+    operations: Vec<u8>,
+}
+
+impl Method {
+    pub fn new(name: &[u8; 4]) -> Self {
```

Comment:
I'd really like us to start adding doc comments for pub methods. This crate is quite old but lets fixup stuff as we add/refactor.